internal/api/token.go

```diff
@@ -433,17 +433,9 @@ func (a *API) updateMFASessionAndClaims(r *http.Request, tx *storage.Connection,
 	var tokenString string
 	var expiresAt int64
 	var refreshToken *models.RefreshToken
-	currentClaims := getClaims(ctx)
-	sessionId, err := uuid.FromString(currentClaims.SessionId)
```

Comment:
Might need to temporarily revert this to get tests to pass

Reply:
Feel free to fix it in this PR.